pennylane/circuit_graph.py

```diff
@@ -676,8 +682,13 @@ def has_path(self, a, b):
 
         return (
             len(
-                rx._digraph_dijkstra_shortest_path(
-                    self._graph, self._graph.nodes().index(a), self._graph.nodes().index(b)
+                rx.digraph_dijkstra_shortest_paths(
```

Comment:
oh so there is a public equivalent?

Reply:
Yes! This is the public equivalent of what I used early on.